### ISSUE TITLE
Butchery Efficiency/Mechanoid Shredding Efficiency capped to 100%

### ISSUE THREAD

**Issue by D9sinc** [Alexis-Grimm/Un-Limited#21]
As the title says I started up my save file, I gave my colonists Bionic Limbs, and thanks to Quality Bionics I was able to make them a lot more efficient than they would've been normally. I noticed when I told my colonist to butcher or shred mechanoids nothing would change compared to before the limbs. I chalked it up to a possible incompatibility with Ore Yields possibly messing with something leading to it so I removed it and I noticed the issue was still persisting. I saw someone had posted a similar issue a few years back which was since closed but it seems to be persisting again. I made sure to check the mod settings and I even tried it with the ButcheryFlesh/Mechanoidefficiency - Max Value setting disabled and enabled for both but neither seems to have resolved the issue.

https://gist.github.com/c009a75613eb65109e8820805a9f7c54 I used Hugslib to see if maybe there is another mod that I could be using that is stoping it from working and if anything else is needed let me know. Thank you for your time and have a lovely day.

**Comment by Alexis-Grimm** [comment]
Ah I see the problem, will have it fixed soon

**Comment by Alexis-Grimm** [comment]
Did you restart your game after enabling ButcheryFlesh/MechanoidEfficiency? I forgot to readd a warning that it needs to be restarted to take effect

**Comment by D9sinc** [comment]
Yes I did. I figured I would restart first because I assumed it would need to be restarted for the change to take effect, but even after a restart the issue was still ongoing.

**Comment by Alexis-Grimm** [comment]
Could you send a picture of the stat in question?

**Comment by D9sinc** [comment]
![20221025013159_1](https://user-images.githubusercontent.com/116627797/197812112-83b1a747-0bb6-46d9-99e6-34a128bc79e4.jpg)
A picture of my setting showing that it's enabled in the mod as well

**Comment by D9sinc** [comment]
![20221025092256_1](https://user-images.githubusercontent.com/116627797/197814986-562bf2dd-3a19-4273-a7d6-2a9fee1d33ef.jpg)

**Comment by Alexis-Grimm** [comment]
Interesting, I'll have to do some more checking but I can't seem to reproduce the issue

**Comment by Alexis-Grimm** [comment]
This should be fixed with 3.1.0
